keysign/discover.py

```diff
@@ -28,13 +28,23 @@ def __init__(self, userdata, discovery):
             self.discovery = AvahiKeysignDiscoveryWithMac()
         self.bt = None
         self.stopped = False
-
+    
     @inlineCallbacks
     def start(self):
         # First we try Avahi, if it fails we fallback to Bluetooth because
         # the receiver may be able to use only one of them
         log.info("Trying to use this code with Avahi: %s", self.userdata)
-        key_data = yield threads.deferToThread(self.discovery.find_key, self.userdata)
+        
+        try:
+            key_data = yield threads.deferToThread(self.discovery.find_key, self.userdata)
+            log.debug("Received key successfully")
+        except ValueError:
+            log.debug('Unable to get key_data, received malformed or altered key')
+            key_data = None
+            success = False
+            message = ""
```

Comment:
Hrm. Can you provide a message here?  Maybe smth like "error downloading key, maybe it has been altered in transit" or so.

Reply:
Sure, I'll add this message.

Reply:
@muelli  Do you want me to pass it in message variable or to log in to console when an exception occurred?

Reply:
I want you to not return an empty message back to the caller. So fill in something reasonable for the message to be retuned.

Reply:
Okay, your suggested message looks good. I'll add that.